tuf/util.py

```diff
@@ -1092,18 +1092,33 @@ def load_file(filepath):
 
 def load_string(data):
   """
-  Loads the given DER or JSON string into TUF's standard Python dictionary
+  Loads the given DER or JSON data into TUF's standard Python dictionary
```

Comment:
We should not get into the habit of omitting doctrings from the functions we create.